src/worker.js

```diff
@@ -331,12 +332,27 @@ self.onmessage = (e) => {
       }
       constraint.uuid = uuid
       state.world.addConstraint(constraint)
+
+      const postStepConstraint = () => {
+        if (isNaN(maxMultiplier)) {
+          return
+        }
+        // The multiplier is proportional to how much force that is added to the bodies by the constraint.
+        // If this exceeds a limit we disable the constraint.
+        const multiplier = Math.abs(constraint.equations[0].multiplier)
+        if (multiplier > maxMultiplier) {
+          constraint.disable()
```

Comment:
I disabled the constraint instead of removing, so it can be renamed.

- [ ] TODO: Not yet implemented would be an event to tell the main thread the worker has disabled the constraint.

Do you think this is a good idea / worthwhile?

Any preference on the API for implementing this event?